pkg/util/exec/exec.go

```diff
@@ -165,7 +165,9 @@ func (*CommandExecutor) ExecuteCommandWithOutputFileTimeout(timeout time.Duratio
 	// if there was anything that went to stdout/stderr then log it, even before
 	// we return an error
 	if string(cmdOut) != "" {
-		logger.Debug(string(cmdOut))
+		if !strings.Contains(err.Error(), "error calling conf_read_file") {
```

Comment:
seems like we would still need the debug logging if the cluster is done being initialized, but maybe it's a corner case we don't need to worry about since it's a very specific error message?

Reply:
Yes, it's very specific to the fact that the operator hasn't written the ceph.conf file yet.